lib/rules/lint-no-invalid-interactive.js

```diff
@@ -155,6 +166,14 @@ function isActionName(node, name) {
   return false;
 }
 
+function isActionName(node, name) {
```

Comment:
Now that I'm reading this diff, I realize that the name of these (preexisting) functions aren't quite correct. We aren't looking for an "action name" we are looking for an event name.

Would you mind cleaning that up a bit? I think this function should be called `isEventHandlerFor`...